presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestStateCompiler.java

```diff
@@ -271,6 +272,33 @@ private long getComplexStateRetainedSize(TestComplexState state)
         return retainedSize;
     }
 
+    private static long getBlockBigArrayReferenceCountMapOverhead(TestComplexState state)
+    {
+        long overhead = 0;
+        Field[] stateFields = state.getClass().getDeclaredFields();
```

Comment:
Add a comment that reflection is necessary because `TestComplexState` implementation is generated.